src/app/components/upload-card/UploadCardRenderer.tsx

```diff
@@ -129,7 +138,19 @@ export function UploadCardRenderer({
       bottom={
         <>
           {fileItem.originalFile.type.startsWith('image') && (
-            <ImagePreview fileItem={fileItem} onSpoiler={handleSpoiler} />
+            <MediaPreview
+              fileItem={fileItem}
+              onSpoiler={handleSpoiler}
+              renderPreviewElement={(p) => <img {...p} alt={p.alt} />}
+            />
+          )}
+          {fileItem.originalFile.type.startsWith('video') && (
+            <MediaPreview
+              fileItem={fileItem}
+              onSpoiler={handleSpoiler}
+              // eslint-disable-next-line jsx-a11y/media-has-caption
+              renderPreviewElement={(p) => <video {...p} />}
```

Comment:
will it be a good idea to allow the video preview to be playable?

Reply:
The preview takes up so little screen space that there wouldn't be room for control widgets beyond a play/pause button, which I decided were out of the scope of this PR.